src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the import path for the `Person` component is correct. Verify that the `Person` component is exported from the `./components/Person/Person` file. If the path or export is incorrect, it will lead to an error when rendering the component.